recipes/andreasbuhr-cppcoro/all/test_package/conanfile.py

```diff
@@ -0,0 +1,16 @@
+import os
+from conans import ConanFile, CMake, tools
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
```

Comment:
For the cmake components in test package

```suggestion
    generators = "cmake", "cmake_find_package_multi"
```